website/blog/2022/02/11/cve-2022-24112.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Apache APISIX Risk Notice for Rewriting X-REAL-IP Header (CVE-2022-24112)"
+keywords: 
+- Risk Notice
+- Header
+- IP restrictions
+- CHAITIN
+description: In versions prior to Apache APISIX 2.12.1, there is a risk of rewriting X-REAL-IP header after enabling the Apache APISIX `batch- requests` plug-in. Now the processing information will be announced.
+tags: [Security]
+---
+
+> In versions prior to Apache APISIX 2.12.1, there is a risk of rewriting X-REAL-IP header after enabling the Apache APISIX `batch- requests` plug-in. Now the processing information will be announced.
+
+<!--truncate-->
+
+## Problem Description
+
+In versions of Apache APISIX prior to 2.12.1 (excluding 2.12.1 and 2.10.4), there is a risk of rewriting the X-REAL-IP header when the Apache APISIX batch-requests plugin is enabled.
+
+This risk leads to two problems:
+
+- An attacker bypasses the IP restrictions on the Apache APISIX data plane via the batch-requests plugin. For example, bypassing IP black and white list restrictions.
+- If the user uses the default Apache APISIX configuration (Admin API enabled, with the default Admin Key and no additional administrative ports assigned), an attacker can invoke the Admin API via the batch-requests plug-in.
```

Comment:
Corresponding to the concept in APISIX, I think it is more appropriate to use admin port here

```suggestion
- If the user uses the default Apache APISIX configuration (Admin API enabled, with the default Admin Key and no additional admin port assigned), an attacker can invoke the Admin API via the batch-requests plug-in.
```